py/kubeflow/testing/cleanup_ci.py

```diff
@@ -579,10 +583,27 @@ def cleanup_health_checks(args):
   logging.info("Matched health checks:\n%s", "\n".join(matched))
   logging.info("Finished cleanup firewall rules")
 
+def get_ssl_certificate_domain(certificate):
```

Comment:
Is this not available via the API?
If we can't get it via the API then we could shell out to gcloud 

```
gcloud beta --project=kubeflow-ci-deployment compute ssl-certificates list --format=yaml
```

and just parse the result

Reply:
Oh I see it looks we are now getting the domain from the certificate in the managed domains field